nginx/timings.py

```diff
@@ -24,11 +24,22 @@
     'cache_status'
 }
 
-STATIC_GROUPS = {
-    '/home/': '/home/',
-    '/pricing/': '/pricing/',
-    '/accounts/login/': 'login'
-}
+# These patterns are to be tried _in order_
+# Group name is given by the `group_name` matching group, with the second element as fallback
+URL_PATTERN_GROUPS = [
+    (re.compile(r'^/a/[^/]*/(?P<group_name>phone/[^/]*)'), None),
+    (re.compile(r'^/a/[^/]*/(?P<group_name>[^/]*)'), None),
```

Comment:
```suggestion
    (re.compile(r'^/a/[^/]+/(?P<group_name>[^/]+)'), None),
```
The same applies to the line above.

Reply:
I'm finding it hard to evaluate if the logic on these two has changed in a significant way. Should `None` be changed to `"other"`? I guess not since there will always be a `group_name` match for these? (thinking out loud a bit there)

What do you think of adding a test that verifies that there is a `(?P<group_name>` substring in the regex pattern if the fallback is `None`?

Reply:
OK I can add such a test or assertion. I'll also make the * => + change

Reply:
My intention was to not change the logic at all. These patterns are fairly well-tested, at least in the most basic way, so I'm fairly certain I didn't break anything

Reply:
Thanks! I am less worried about you having made breaking changes in this PR than I am about someone coming along in the future and adding a bad configuration (e.g., regex without `group_name` and default of `None`) because they didn't realize the interdependency. Hence the request for the extra test.